code/datums/components/acid.dm

```diff
@@ -190,24 +202,25 @@ GLOBAL_DATUM_INIT(acid_overlay, /mutable_appearance, mutable_appearance('icons/e
 	return NONE
 
 /// Handles searing the hand of anyone who tries to touch this without protection.
-/datum/component/acid/proc/on_attack_hand(atom/parent_atom, mob/living/carbon/user)
+/datum/component/acid/proc/on_attack_hand(atom/source, mob/living/carbon/user)
 	SIGNAL_HANDLER
 
 	if(!istype(user))
 		return NONE
-	if((parent_atom == user) || (parent_atom.loc == user))
+	if((source == user) || (source.loc == user))
 		return NONE // So people can take their own clothes off.
 	if((acid_power * acid_volume) < ACID_LEVEL_HANDBURN)
 		return NONE
-	if(user.gloves?.resistance_flags & (UNACIDABLE|ACID_PROOF))
+	if(user.gloves?.resistance_flags & (UNACIDABLE | ACID_PROOF))
```

Comment:
FLAG|FLAG is fine syntax for the record. I don't mind all that much here but you shouldn't change it if you find many instances in a large file